### DISCUSSION TITLE
Can`t connect to ollama

### DISCUSSION THREAD

**Discussion by gitttyyy** [danny-avila/LibreChat#2264] [Troubleshooting] [answered]
![image](https://github.com/danny-avila/LibreChat/assets/154318356/6dd77b40-34e1-4679-878e-1842b13fee9b)
![image](https://github.com/danny-avila/LibreChat/assets/154318356/7fa6f1ee-bc3a-4e35-a00f-7837906acd61)
Any wrong with my config? I just follow the guide and all the port is the default

**Comment by danny-avila** [comment]
Try using "0.0.0.0" instead of "127.0.0.1"

You can also try entering the container terminal and using curl to ping ollama to see what works first.

**Reply by gitttyyy** [reply, marked as answer]
My fault, I used docker to run librechat, and then the host ran ollama. I should have changed it to `host.docker.internal`.
